### DISCUSSION TITLE
Basic media player - how setup ?

### DISCUSSION THREAD

**Discussion by zlomar** [music-assistant/support#1079] [Q&A] [answered]
Hello, 

I configured MA correct at your instruction , but I have a question ?
How configure basic mediaplayer ? 
When I tell OK Google - play music - the music start on media_player.xxxxx not media_player.mass_xxxxx  ? 

I have a "hided" of media_player.xxxxx

**Comment by OzGav** [comment]
Are you using HA cloud?

**Reply by zlomar** [reply]
Means nabu casa? No i don't.

**Reply by OzGav** [reply, marked as answer]
Then I dont think Google has visibility of the MA player then does it?
